### PR TITLE
Fixes issue #7

### LINKED ISSUE #7: invalid integer with new fishshell version
With the new version of fish `version 2.4.0-291-g7211802f` I see the following error

```
test returned eval errors:
	invalid integer ''
```

With `fish -d 3` I see

```
<3> fish: Job is constructed
<3> fish: proc::read_try('set -l status_color 0fc')

<3> fish: Skipping fork: no output for internal builtin 'test'
<3> fish: Set status of test "$status_copy" -ne 0 to 1 using short circuit
<3> fish: Job is constructed
<3> fish: proc::read_try('test "$status_copy" -ne 0')

test returned eval errors:
	invalid integer ''
```

Guessing the culprit is `test "$status_copy" -ne 0` 

Any ideas what is causing this?

### PR DESCRIPTION
**Pull request by amir20**
I think this fixes #7 . But someone else should validate.

### CODE REVIEW

**Comment by jbucaran** [comment]
@amir20 Thanks :)

Can you make it like so:

```fish
if test 0"$CMD_DURATION" -gt 100
...
end
```

It should work too, but please make sure it does indeed.

File: fish_right_prompt.fish [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
         set status_color $fish_color_error
     end
 
-    if test "$CMD_DURATION" -gt 100
-        set -l duration_copy $CMD_DURATION
-        set -l duration (echo $CMD_DURATION | humanize_duration)
-
-        echo -sn (set_color $status_color) "$duration" (set_color normal)
+    if test $CMD_DURATION
+      if test "$CMD_DURATION" -gt 100
```
Comment (faho) [inline fish_right_prompt.fish]: Personally, I'd either put this in one test call (`test -n "$CMD_DURATION" -a "$CMD_DURATION" -gt 100`) or use `set -q CMD_DURATION; and test "$CMD_DURATION" -gt 100`.

Both save one block-level.

File: fish_right_prompt.fish [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
+          set -l duration_copy $CMD_DURATION
+          set -l duration (echo $CMD_DURATION | humanize_duration)
```
Comment (faho) [inline fish_right_prompt.fish]: Why does `duration_copy` exist if it's not being used then?

File: fish_right_prompt.fish (hunk 1 above)
Comment (jbucaran) [inline fish_right_prompt.fish]: Yes, that would work too 👍

File: fish_right_prompt.fish (hunk 2 above)
Comment (jbucaran) [inline fish_right_prompt.fish]: 🤔 That must be a leftover from a different prompt that was copied into this one by mistake. It clearly seems redundant.

Comment (jbucaran) [inline fish_right_prompt.fish]: https://github.com/search?q=org%3Afisherman+duration_copy&type=Code

LOL

**Comment by amir20** [comment]
All comments have been addressed. Merging.

**Pull request merged**